src/content/reference/docker-compose.mdx

```diff
@@ -563,3 +563,34 @@ endpoints:
       service: backend
       port: 8080
 ```
+
+### update (annotation, optional)
+
+You can set the update policy of a service by setting the annotation `dev.okteto.com/update`. 
+
+Depending on the kind of resource the acceptance values are different:
+- rolling: Will deploy a service with 0 downtime. Allowed on deployments and statefulsets.
+- recreate: Will wait until the current pod is deleted to start creating the new one. Allowed on deployments
+- on-delete: Will wait until the current pod is deleted to start creating the new one. Allowed on statefulset
+
+```yaml
+services:
+  deployment:
+    image: python:alpine
+    labels:
```

Comment:
@jLopezbarb were there an environment variable to define this behavior too?

Also, I am wondering if this would be a better fit for the `x-okteto-update` notation

Reply:
Agree with @pchico83 , this should follow the x-okteto since that's what've been using for others. I'd go with `x-okteto-update-policy`